config.yaml

```diff
@@ -26,10 +26,10 @@ params:
   footer:
     sections:
       - type: footer_text
-        image: images/logo-alt.svg
+        image: images/tardis-banner-2.svg
```

Comment:
You need to constraint the width of this banner (in corresponding sass or css file) it's otherwise occupying entire width of column and is very large. Let's go with `width: 180px;` since the azimuth logo had 162 px